devfile.yaml

```
@@ -0,0 +1,40 @@
+schemaVersion: 2.1.0
+metadata:
+  name: vscode-devfile
+components:
+  - name: tools
+    container:
+      image: quay.io/devfile/universal-developer-image:ubi8-latest
+      memoryLimit: 4G
```

It's better to specify all resources

```suggestion
      memoryRequest: 1G
      memoryLimit: 4G
      cpuRequest: '1'
      cpuLimit: '2'
```